### PR TITLE
Update the design of the landing page

### PR DESCRIPTION
**Pull request by dcoscolla-intermedia**
### 📑 Changes:

localhost_3000_ (3)

- Refactor landing page layout
- Refactor responsive styles
- Tweak spacing and paddings
- Tweak font sizes
- Add a Downloads Section
- Home download button react to current user navigator

### ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: webpage/src/utils/getBrowser.ts
Comment (jmartel07) [inline webpage/src/utils/getBrowser.ts:1]: @bcarpintero-intermediait can we schedule a call to review the process to move this code to our custom lib? cc @dcoscolla-intermedia @aferrentinointermedia

**Review by jmartel07 — APPROVED** [review]

File: webpage/src/app/(landing)/components/download-button.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+'use client';
+
+import { BROWSERS, getBrowser } from '@/utils/getBrowser';
+import { Button, Link } from '@chakra-ui/react';
+import NextLink from 'next/link';
+import { ReactElement, useEffect, useState } from 'react';
+import { FaChrome, FaFirefox, FaSafari } from 'react-icons/fa6';
+
+interface DownloadButtonProps {}
```
Comment (aferrentinointermedia) [inline webpage/src/app/(landing)/components/download-button.tsx:9]: Esta bien que esta interfaz quede vacía?

File: webpage/src/utils/getBrowser.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export function getBrowser() {
+  if (typeof navigator === 'undefined') {
+    return BROWSERS.UNKNOWN;
+  }
+
+  const userAgent = navigator.userAgent;
+  let browserName = BROWSERS.UNKNOWN;
+
+  if (userAgent.indexOf('Firefox') > -1) {
+    browserName = BROWSERS.FIREFOX;
```
Comment (aferrentinointermedia) [inline webpage/src/utils/getBrowser.ts:21]: No se puede hacer un .includes("Firefox") ponele? Capaz se podría hacer un switch

Comment (aferrentinointermedia) [reply webpage/src/utils/getBrowser.ts:21]: Creo que el .includes directamente devuelve true o false. Tambien, podríamos meter un diccionario donde tengas las keys por lo que buscas, algo asi:
`"Firefox":BROWSERS.FIREFOX`

**Review by aferrentinointermedia — CHANGES_REQUESTED** [review]

File: webpage/src/utils/getBrowser.ts (hunk 2 above)
Comment (aferrentinointermedia) [reply webpage/src/utils/getBrowser.ts:21]: Algo asi me refiero>
image

File: webpage/src/app/(landing)/components/download-button.tsx (hunk 1 above)
Comment (dcoscolla-intermedia) [reply webpage/src/app/(landing)/components/download-button.tsx:9]: This shouldn’t cause any problems

**Pull request merged**